### PR TITLE
Updated privacy controls in application setup

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?

Describes a new config set up option for automatically collecting user data.

### Motivation

[DOCS-4882](https://datadoghq.atlassian.net/browse/DOCS-4882), which is a part of [DOCS-4827](https://datadoghq.atlassian.net/browse/DOCS-4827).

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

[DOCS-4882]: https://datadoghq.atlassian.net/browse/DOCS-4882?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DOCS-4827]: https://datadoghq.atlassian.net/browse/DOCS-4827?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/rum_getting_started.md [hunk 1]
```diff
@@ -373,3 +374,4 @@ Add the following snippet during initialization in your Datadog configuration:
 [12]: https://github.com/DataDog/dd-sdk-android-gradle-plugin
 [13]: https://docs.datadoghq.com/real_user_monitoring/android/web_view_tracking/
 [14]: https://docs.datadoghq.com/getting_started/tagging/using_tags/#rum--session-replay
+[15]: /real_user_monitoring/android/data_collected/
```
Comment (0xnm) [inline docs/rum_getting_started.md:377]: should this has the same format as links above? with full URL?

File: docs/rum_getting_started.md [hunk 2]
```diff
@@ -42,8 +42,9 @@ dependencies {
 1. Navigate to [**UX Monitoring** > **RUM Applications** > **New Application**][2].
 2. Select `android` as the application type and enter an application name to generate a unique Datadog application ID and client token.
 3. To instrument your web views, click the **Instrument your webviews** toggle. For more information, see [Web View Tracking][13].
+4. To disable automatic user data collection for either client IP or geolocation data, uncheck the boxes for those settings. For more information, see [RUM Android Data Collected][15].
 
-{{< img src="real_user_monitoring/android/create_rum_application.png" alt="Create a RUM application in Datadog workflow" style="width:90%;">}}
+   {{< img src="real_user_monitoring/android/new-rum-app-android-1.png" alt="Create a RUM application in Datadog workflow" style="width:90%;">}}
```
Comment (0xnm) [inline docs/rum_getting_started.md:47]: I get 404 when trying to open it using https://docs.datadoghq.com/real_user_monitoring/android/new-rum-app-android-1.png. The asset is not yet uploaded?

File: docs/rum_getting_started.md (hunk 1 above)
Comment (rtrieu) [reply docs/rum_getting_started.md]: Thanks for this call out!

File: docs/rum_getting_started.md (hunk 2 above)
Comment (buraizu) [inline docs/rum_getting_started.md:47]:
```suggestion
   {{< img src="real_user_monitoring/android/new-rum-app-android-1.png" alt="Create a RUM application for Android in Datadog" style="width:90%;">}}
```

Comment (buraizu) [reply docs/rum_getting_started.md:47]: Just an optional suggestion here to match the style on the other RUM pages

**Review by buraizu — APPROVED** [review]
LGTM, left one optional suggestion

**Review by 0xnm — APPROVED** [review]

**Review by xgouchet — APPROVED** [review]

**Pull request merged**